ui/src/cloud/apis/demodata.ts

```diff
@@ -1,56 +1,49 @@
 // Libraries
 import {get} from 'lodash'
-import {getBuckets} from 'src/client'
+import {getBuckets, getBucket} from 'src/client'
 import AJAX from 'src/utils/ajax'
 
 //Utils
 import {isFlagEnabled} from 'src/shared/utils/featureFlag'
 
 //Types
-import {Bucket, DemoBucket} from 'src/types'
+import {Bucket, DemoBucket, BucketEntities} from 'src/types'
 import {LIMIT} from 'src/resources/constants'
+import {normalize} from 'normalizr'
+import {bucketSchema} from 'src/schemas'
+import {NormalizedSchema} from 'normalizr'
 
 const baseURL = '/api/v2/experimental/sampledata'
 
 export const getDemoDataBuckets = async (): Promise<Bucket[]> => {
-  try {
-    const {data} = await AJAX({
-      method: 'GET',
-      url: `${baseURL}/buckets`,
-    })
-
-    // if sampledata endpoints are not available in a cluster
-    // gateway responds with a list of links where 'buckets' field is a string
-    const buckets = get(data, 'buckets', false)
-    if (!Array.isArray(buckets)) {
-      throw new Error('Could not reach demodata endpoint')
-    }
-
-    return buckets.filter(b => b.type == 'user') as Bucket[] // remove returned _tasks and _monitoring buckets
-  } catch (error) {
-    console.error(error)
-    throw error
+  const {data} = await AJAX({
+    method: 'GET',
+    url: `${baseURL}/buckets`,
+  })
+
+  // if sampledata endpoints are not available in a cluster
+  // gateway responds with a list of links where 'buckets' field is a string
+  const buckets = get(data, 'buckets', false)
```

Comment:
nit: `null` fits this better conceptually than `false`. `null` generally means has 'no value' and applies a little bit more cleanly than `false` does to an array. they both accomplish the same thing, it's just a little bit more conceptually sane to assign `null` to a collection than `false`.